src/patternfly/components/Tabs/tabs.scss

```diff
@@ -300,22 +319,23 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
   flex: 1;
   padding: var(--pf-c-tabs__link--PaddingTop) var(--pf-c-tabs__link--PaddingRight) var(--pf-c-tabs__link--PaddingBottom) var(--pf-c-tabs__link--PaddingLeft);
   color: var(--pf-c-tabs__link--Color);
-  user-select: none;
   background-color: var(--pf-c-tabs__link--BackgroundColor);
   outline-offset: var(--pf-c-tabs__link--OutlineOffset);
 
   &::before {
+    pointer-events: none;
     border-color: var(--pf-c-tabs__link--before--BorderColor);
-    border-style: var(--pf-c-tabs__link--before--BorderStyle);
     border-width: var(--pf-c-tabs__link--before--BorderTopWidth) var(--pf-c-tabs__link--before--BorderRightWidth) var(--pf-c-tabs__link--before--BorderBottomWidth) var(--pf-c-tabs__link--before--BorderLeftWidth);
     border-right-color: var(--pf-c-tabs__link--before--BorderRightColor);
     border-bottom-color: var(--pf-c-tabs__link--before--BorderBottomColor);
   }
 
   &::after {
+    top: var(--pf-c-tabs__link--after--Top);
+    right: var(--pf-c-tabs__link--after--Right);
+    bottom: var(--pf-c-tabs__link--after--Bottom);
     left: var(--pf-c-tabs__link--before--Left);
```

Comment:
should this use the variable for `::after`?

Reply:
Nice catch! I annotated its purpose.